.circleci/config.yml

```diff
@@ -35,6 +35,37 @@ defaults: &defaults
   working_directory: ~/repo
 
 jobs:
+  build_elixir_1_11_otp_23:
+    docker:
+      - image: erlang:23.1.3
+        environment:
+          ELIXIR_VERSION: 1.11.2-otp-23
```

Comment:
Caching and dialyzer need to be removed for the elixir 1.10 build or there's a race on the ci cache.

Reply:
Fixed in #51